### PR TITLE
Added job to run SKR fast-integration tests as pre-submit for eventing

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**
- Job to run SKR fast-integration tests as pre-submit for eventing

**Related PR(s)**
- https://github.com/kyma-project/kyma/pull/12882

**Related issue(s)**

- https://github.com/kyma-project/kyma/issues/12848

### CODE REVIEW

File: templates/data/kyma-integration-gardener-data.yaml [hunk 1]
```diff
@@ -93,6 +107,22 @@ templates:
...
             annotations:
               pipeline.platform: gardener_gcp
+          skr_aws_job_eventing:
+            labels:
+              preset-cluster-version: "true"
+            env:
+              KYMA_PROJECT_DIR: "/home/prow/go/src/github.com/kyma-project"
+              CREDENTIALS_DIR: "/etc/credentials/kyma-tunas-prow-event-mesh"
+              # KEB_PLAN_ID: "361c511f-f939-4621-b228-d0fb79a1fe15" # AWS
+              # KEB_PLAN_ID: "5cb3d976-b85c-42ea-a636-79cadda109a9" # Preview Plan
```
Comment (marcobebway) [inline templates/data/kyma-integration-gardener-data.yaml:117]: Do we need those comments?

**Review by marcobebway — APPROVED** [review]

File: templates/data/kyma-integration-gardener-data.yaml (hunk 1 above)
Comment (mfaizanse) [reply templates/data/kyma-integration-gardener-data.yaml:117]: it would be helpful for later if to switch to different plan.

**Review by marcobebway — DISMISSED** [review]

File: templates/data/kyma-integration-gardener-data.yaml [hunk 2]
```diff
@@ -42,6 +42,20 @@ templates:
               pipeline.test: "fast-integration"
               pipeline.clusterprovisioning: kyma cli
             command: "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/kyma-integration-gardener-eventing.sh"
+          command_integration_skr_eventing:
+            cluster: "trusted-workload"
```
Comment (Halamix2) [inline templates/data/kyma-integration-gardener-data.yaml:46]: Does this job need to run on trusted cluster?

**Review by Halamix2 — CHANGES_REQUESTED** [review]

File: templates/data/kyma-integration-gardener-data.yaml (hunk 2 above)
Comment (mfaizanse) [reply templates/data/kyma-integration-gardener-data.yaml]: updated

**Review by Halamix2 — APPROVED** [review]

**Pull request merged**